### ISSUE TITLE
🔬 SciML — 2026-04-04 (22 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#44]
### ⭐ Spotlight

**1. [Learning Boltzmann Generators via Constrained Mass Transport](https://openreview.net/forum?id=KCIbELeuK0)**
  *Christopher von Klitzing, Denis Blessing, Henrik Schopmans et al.*
  📍 NeurIPS2025-AI4Science Spotlight
  > Efficient sampling from high-dimensional and multimodal unnormalized probability distributions is a central challenge in many areas of science and machine learning. We focus on Boltzmann generators...

### 📌 Poster

**2. [Common Task Framework For a Critical Evaluation of Scientific Machine Learning Algorithms](https://openreview.net/forum?id=pkFKjmUE3L)**
  *Philippe Martin Wyder, Judah A Goldfeder, Alexey Yermakov et al.*
  📍 NeurIPS 2025 Datasets and Benchmarks Track poster
  > Machine learning (ML) is transforming modeling and control in the physical, engineering, and biological sciences. However, rapid development has outpaced the creation of standardized, objective ben...

**3. [Towards Foundation Models for Scientific Machine Learning: Characterizing Scaling and Transfer Behavior](https://openreview.net/forum?id=zANxvzflMl)**
  *Shashank Subramanian, Peter Harrington, Kurt Keutzer et al.*
  📍 NeurIPS 2023 poster
  > Pre-trained machine learning (ML) models have shown great performance for a
wide range of applications, in particular in natural language processing (NLP)
and computer vision (CV). Here, we study h...

**4. [Scientific Machine Learning for Symbolic Recovery of Relativistic Effects in Black Hole Orbits](https://openreview.net/forum?id=pcUoDfa4Yd)**
  *Pothuraju Naveen Yadav, Prathamesh Dinesh Joshi, Raj Dandekar et al.*
  📍 NeurIPS2025-AI4Science Poster
  > Simulating relativistic orbital dynamics around Schwarzschild black holes is essential for understanding general relativity and astrophysical phenomena like precession. Traditional numerical solver...

**5. [The Darwin–Gödel Discovery Machine: Toward Bounded-Risk Self-Improving AI4Science](https://openreview.net/forum?id=rgAIYyXzww)**
  *Xuening Wu, Xinhang Zhang, Yanlan Kang et al.*
  📍 NeurIPS2025-AI4Science Poster
  > We present the Darwin–Gödel Discovery Machine (DGDM), a dual-loop system for risk-aware self-improving AI4Science. The inner Darwinian loop evolves candidate solutions—demonstrated here with molecu...

**6. [AI for Open Science: A Multi-Agent Perspective for Ethically Translating Data to Knowledge](https://openreview.net/forum?id=vhEUYsNrGa)**
  *Chase Yakaboski, Gregory Hyde, Clement Nyanhongo et al.*
  📍 NeurIPS2023-AI4Science Poster
  > AI for Science (AI4Science), particularly in the form of self-driving labs, has the potential to sideline human involvement and hinder scientific discovery within the broader community. While prior...

**7. [Spectrum-Informed Multistage Neural Network: Multiscale Function Approximator of Machine Precision](https://openreview.net/forum?id=59hdCpJyHx)**
  *Jakin Ng, Yongji Wang, Ching-Yao Lai*
  📍 ICML2024-AI4Science Poster
  > Deep learning frameworks have become powerful tools for approaching scientific problems such as turbulent flow, which has wide-ranging applications. In practice, however, existing scientific machin...

**8. [PDEfuncta: Spectrally-Aware Neural Representation for PDE Solution Modeling](https://openreview.net/forum?id=NfBrMDF0Xi)**
  *Minju Jo, Woojin Cho, Uvini Balasuriya Mudiyanselage et al.*
  📍 NeurIPS 2025 poster
  > Scientific machine learning often involves representing complex solution fields that exhibit high-frequency features such as sharp transitions, fine-scale oscillations, and localized structures. Wh...

**9. [PEAR: Equal Area Weather Forecasting on the Sphere](https://openreview.net/forum?id=MlnM8lvFSq)**
  *Hampus Linander, Christoffer Petersson, Daniel Persson et al.*
  📍 NeurIPS2025-AI4Science Poster
  > Artificial intelligence is rapidly reshaping the natural sciences, with weather forecasting emerging as a flagship AI4Science application where machine learning models can now rival and even surpas...

**10. [Text2Decision: Decoding Latent Variables in Risky Decision Making from Think Aloud Text](https://openreview.net/forum?id=fEoemPDicz)**
  *Hanbo Xie, Huadong Xiong, Robert C. Wilson*
  📍 NeurIPS2023-AI4Science Poster
  > Understanding human thoughts can be difficult, as scientists usually rely on observing behaviors. The think-aloud protocol, where people talk about their thoughts while making decisions, provides a...

**11. [TokMan:Tokenize Manhattan Mask Optimization for Inverse Lithography](https://openreview.net/forum?id=hsf4gi5iEN)**
  *Yiwen Wu, Yuyang Chen, Ye Xia et al.*
  📍 NeurIPS 2025
  > Manhattan representations, defined by axis-aligned, orthogonal structures, are widely used in vision, robotics, and semiconductor design for their geometric regularity and algorithmic simplicity. I...

### 🆕 Recent (ArXiv / OpenReview)

**12. [Scientific Machine Learning Benchmarks](https://openreview.net/forum?id=J4Hy8UnpPl)**
  *Jeyan Thiyagalingam, Mallikarjun Shankar, Geoffrey C. Fox et al.*
  📍 CoRR 2021
  > The breakthrough in Deep Learning neural networks has transformed the use of AI and machine learning technologies for the analysis of very large experimental datasets. These datasets are typically ...

**13. [Mechanistic Neural Networks for Scientific Machine Learning](https://openreview.net/forum?id=6TR7dg23YF)**
  *Adeel Pervez, Francesco Locatello, Efstratios Gavves*
  📍 CoRR 2024
  > This paper presents Mechanistic Neural Networks, a neural network design for machine learning applications in the sciences. It incorporates a new Mechanistic Block in standard architectures to expl...

**14. [Verification and Validation for Trustworthy Scientific Machine Learning](https://openreview.net/forum?id=TdsfHEhzuw)**
  *John D. Jakeman, Lorena A. Barba, Joaquim R. R. A. Martins et al.*
  📍 CoRR 2025
  > Scientific machine learning (SciML) models are transforming many scientific disciplines. However, the development of good modeling practices to increase the trustworthiness of SciML has lagged behi...

**15. [Overview of ACM SIGKDD 2024 AI4Science4AI Special Day](https://openreview.net/forum?id=3gbzJPV1qe)**
  *Wei Ding, Gustau Camps-Valls*
  📍 KDD 2024
  > This paper provides an overview of the ACM SIGKDD 2024 AI4Science4AI special day. It includes information about the organizers, invited speakers, keynote speakers, the event agenda, and insights fr...

**16. [Bridging AI and Science: Implications from a Large-Scale Literature Analysis of AI4Science](https://openreview.net/forum?id=uDSIPHgDvI)**
  *Yutong Xie, Yijun Pan, Hua Xu et al.*
  📍 CoRR 2024
  > Artificial Intelligence has proven to be a transformative tool for advancing scientific research across a wide range of disciplines. However, a significant gap still exists between AI and scientifi...

**17. [A nice theoretical framework which brings open science to ai4science. However, no concrete steps are proposed to fix ethical issues.](https://openreview.net/forum?id=xm2XYezpLR)**
  **
  > 

**18. [Unlocking the Potential of AI Researchers in Scientific Discovery: What Is Missing?](https://openreview.net/forum?id=z807E7nlcR)**
  *Hengjie Yu, Yaochu Jin*
  📍 CoRR 2025
  > The potential of AI researchers in scientific discovery remains largely untapped. Over the past decade, AI for Science (AI4Science) publications in 145 Nature Index journals have increased fifteen-...

**19. [Artificial Intelligence for Science in Quantum, Atomistic, and Continuum Systems](https://openreview.net/forum?id=9gfQMGh33a)**
  *Xuan Zhang, Limei Wang, Jacob Helwig et al.*
  📍 CoRR 2023
  > Advances in artificial intelligence (AI) are fueling a new paradigm of discoveries in natural sciences. Today, AI has started to advance natural sciences by improving, accelerating, and enabling ou...

**20. [Compact Runge-Kutta flux reconstruction methods with entropy and/or kinetic energy preserving fluxes](http://arxiv.org/abs/2604.02125v1)**
  *Arpit Babbar, Qifan Chen, Hendrik Ranocha*
  > Compact Runge-Kutta (cRK) methods are a class of high order methods for solving hyperbolic conservation laws characterized by their compact stencil including only immediate neighboring finite eleme...

**21. [Graph coarsening: From scientific computing to machine learning](https://openreview.net/forum?id=qH2NkVHQRW)**
  *Jie Chen, Yousef Saad, Zechen Zhang*
  > The general method of graph coarsening or graph reduction has been a remarkably useful and ubiquitous tool in scientific computing and it is now just starting to have a similar impact in machine le...

**22. [A Differentiable Programming System to Bridge Machine Learning and Scientific Computing](https://openreview.net/forum?id=u5W9rpksyf)**
  *Mike Innes, Alan Edelman, Keno Fischer et al.*
  📍 CoRR 2019
  > Scientific computing is increasingly incorporating the advancements in machine learning and the ability to work with large amounts of data. At the same time, machine learning models are becoming in...
